### DISCUSSION TITLE
Error adding new series

### DISCUSSION THREAD

**Discussion by sigjas** [jef/streetmerchant#2194] [Help]
![image](https://user-images.githubusercontent.com/36080853/111876887-47405500-8966-11eb-94e7-6b9034d51ac8.png)

**Comment by My-Name-Is-Adam** [comment]
Also facing this, did you find any fix?

**Comment by 4ut0m4t3** [comment]
Ensure you have created a discord notify group for your new series, inside the dotenv file

ie

DISCORD_NOTIFY_GROUP_TEST=
